nginx.conf

```diff
@@ -124,14 +124,17 @@ http {
             rewrite (?i)^/explained/?$ https://www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=explained&utm_campaign=unfck&utm_content=podcast permanent;
             rewrite (?i)^/wtf/?$ https://www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=wtf&utm_campaign=unfck&utm_content=podcast permanent;
 
+            # issue 32
+            rewrite (?i)^/liebe/?$ https://www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=prosieben&utm_campaign=unfck&utm_content=podcast permanent;
```

Comment:
Is it true that these should be the query params even if there are incoming params? The section below where it was removed keeps incoming params if they exist.

Reply:
Yeah, I double checked on this and the only place this vanity URL is being used are audio ads / podcasts, so I would expect this will always be direct traffic at origin.